src/Symfony/Component/Notifier/Bridge/AmazonSns/AmazonSnsOptions.php

```diff
@@ -0,0 +1,127 @@
+<?php
+
+/*
+ * This file is part of the Symfony package.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Symfony\Component\Notifier\Bridge\AmazonSns;
+
+use AsyncAws\Sns\Input\PublishInput;
+use AsyncAws\Sns\ValueObject\MessageAttributeValue;
+use Symfony\Component\Notifier\Message\MessageOptionsInterface;
+
+/**
+ * @author Adrien Chinour <github@chinour.fr>
+ *
+ * @experimental in 5.3
+ */
+final class AmazonSnsOptions implements MessageOptionsInterface
+{
+    private $options = [];
+
+    public function __construct(array $options = [])
+    {
+        $this->options = $options;
+    }
+
+    public function toArray(): array
+    {
+        $options = $this->options;
+        unset($options['recipient_id']);
+
+        return $options;
+    }
+
+    public function getRecipientId(): ?string
+    {
+        return $this->options['recipient_id'] ?? null;
+    }
+
+    /**
+     * @param string $topic The Topic ARN for SNS message
+     *
+     * @return $this
+     */
+    public function recipient(string $topic): self
+    {
+        $this->options['recipient_id'] = $topic;
+
+        return $this;
+    }
+
+    /**
+     * @param string $subject
```

Comment:
To be removed, we to not add phpdocs when they don't add any useful information that is not already in the PHP signature (I let you remove the other phpdocs that can be removed for the same reason)